lib/portlayer/exec/container.go

```diff
@@ -451,8 +452,16 @@ func (c *Container) Remove(ctx context.Context, sess *session.Session) error {
 		return c.vm.DeleteExceptDisks(ctx)
 	})
 	if err != nil {
-		c.State = existingState
-		return err
+		if f, ok := err.(types.HasFault); ok {
+			switch f.Fault().(type) {
```

Comment:
can we have a `default:` target that logs the fault type - currently we lose it completely.